demo/components/Playground.tsx

```diff
@@ -77,6 +77,7 @@ export type PlaygroundProps = {
     allowHTML?: boolean;
     settingsVisible?: boolean;
     initialEditor?: MarkdownEditorMode;
+    preserveEmptyRows?: boolean;
```

Comment:
Add preserveEmptyRows to the following [line](https://github.com/gravity-ui/markdown-editor/blob/b536dc054790900eafd3997a8d7e88fdd396163a/demo/components/Playground.tsx#L239) in Playground.tsx to enable rerendering in the demo.